kube-monitoring/plugindefinition.yaml

```diff
@@ -14,7 +14,7 @@ spec:
   helmChart:
     name: kube-monitoring
     repository: oci://ghcr.io/cloudoperators/greenhouse-extensions/charts
-    version: 0.20.1
+    version: 0.21.0
```

Comment:
Please bump the PluginDefinition version as well here - https://github.com/cloudoperators/greenhouse-extensions/blob/53aa9fc766274a8a70eb0d0cd9a17d39b247c164/kube-monitoring/plugindefinition.yaml#L9